test/generators/java/__snapshots__/JavaGenerator.spec.ts.snap

```diff
@@ -1052,6 +1052,18 @@ exports[`JavaGenerator should render enums with translated special characters 1`
 exports[`JavaGenerator should render models and their dependencies 1`] = `
 Array [
   "package test.packageName;
+
+
+",
+  "package test.packageName;
+
+
+",
+  "package test.packageName;
+
+
+",
```

Comment:
This seems a bit wrong 🤨

Reply:
Well, it's not introduced in this PR. The order of the output has changed a bit because of the changes in `AbstractGenerator.getConstrainedModels`.